### PR TITLE
Create integrations-and-incident-health-check.md

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/29954

## Description
Added a new article for the Integrations and incidents health check, this article explains more about how to configure and use the pack contents.

### CODE REVIEW

File: content-repo/extra-docs/articles/integrations-and-incident-health-check.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ 
+### Dashboards and Lists
+The dashboard widgets use the data from the lists created or updated by the playbook.
+The dashboard **isn't** a real-time view **The dashboard displays are the data for the time the playbook was running.**
+For example, if the job/playbook runs once a day, the dashboard data will be updated once a day (at the same time the playbook is running).
+ 
+## Before You Start
+ 
+The playbooks in the pack use two scripts to check the integrations and incidents. 
+"Demisto REST API" integration is mandatory for one of the scripts to run.
```
Comment (michalgold) [inline content-repo/extra-docs/articles/integrations-and-incident-health-check.md:44]: The Demisto REST API pack will be automatically installed with this pack if not already installed.

**Comment by kirbles19** [comment]
@altmannyarden Updated.

File: content-repo/extra-docs/articles/integrations-and-incident-health-check.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+3. **(IMPORTANT)** Copy and save the API key, you will not be able to access it again.
+4. Go to **Settings > INTEGRATIONS > Servers & Services** and search for **Demisto REST API**.
+5. Click **Add instance** and enter the required information.
+    - A meaningful name for the integration instance
+    - Demisto Server URL
+    - API key that you generated
+7. Click the **Test** button to make sure that that server and API key are reachable and valid.
+8. Click **Done**.
+ 
+ ### 2. Configure a Mail Sender and Mail Listener Integration
```
Comment (michalgold) [inline content-repo/extra-docs/articles/integrations-and-incident-health-check.md:81]: this is optional though, not a mandatory task. @altmannyarden @kirbles19

File: content-repo/extra-docs/articles/integrations-and-incident-health-check.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+| JOB - Integrations and Playbooks Health Check - Lists handling | This playbook is executed as part of the **JOB - Integrations and Playbooks Health** parent playbook and is responsible for creating or updating related XSOAR lists. | - |
+ ---
+ 
+In addition to running the sub-playbooks, the parent playbook **JOB - Integrations and Playbooks Health Check** performs the following functions:
+- Send the health check report to the recipients from the playbook inputs.
+- If another health check investigation is open (incident, not a job), the playbook will:
+  - Check if analyst notes were added to the failed integrations and failed incidents grids and **copy** them to the new investigation.
+  - Link the old investigation to the new one and close the old investigation due to irrelevance.
+Re-run the health check tests. It is a manual action because you will want to re-run them only after fixing the issues, not before.
```
Comment (altmannyarden) [inline content-repo/extra-docs/articles/integrations-and-incident-health-check.md:31]: i think we should add "-" before the "Re-run the health check tests. It is a manual.... "

File: content-repo/extra-docs/articles/integrations-and-incident-health-check.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+ 
+### Incident Layouts
+The incident type contains two layouts: one for ***failed integration instances** and one for **failed incidents**.
```
Comment (altmannyarden) [inline content-repo/extra-docs/articles/integrations-and-incident-health-check.md:34]: Remove 1 "*" before the failed integration

File: content-repo/extra-docs/articles/integrations-and-incident-health-check.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+| Integrations and Playbooks Health Check - Running Scripts | This playbook is executed as part of the **JOB - Integrations and Playbooks Health** parent playbook and is responsible for running scripts that check the system for failed integrations and failed incidents.  | You can run the playbook on its own (not only as a sub-playbook of the parent playbook) to run health checks on enabled integrations and open incidents. |
+| JOB - Integrations and Playbooks Health Check - Lists handling | This playbook is executed as part of the **JOB - Integrations and Playbooks Health** parent playbook and is responsible for creating or updating related XSOAR lists. | - |
+ ---
+ 
+In addition to running the sub-playbooks, the parent playbook **JOB - Integrations and Playbooks Health Check** performs the following functions:
+- Send the health check report to the recipients from the playbook inputs.
+- If another health check investigation is open (incident, not a job), the playbook will:
```
Comment (altmannyarden) [inline content-repo/extra-docs/articles/integrations-and-incident-health-check.md:28]: also a job (when running a job - a new investigation open too, like incidents)

**Comment by glicht** [comment]
@altmannyarden what is the status here?

**Review by michalgold — APPROVED** [review]

**Pull request merged**